care/facility/api/serializers/bed.py

```diff
@@ -179,10 +179,18 @@ def create(self, validated_data):
         if occupied_beds.filter(bed=bed).exists():
             raise ValidationError({"bed:": ["Bed already in use by patient"]})
 
+        Bed.objects.filter(
```

Comment:
now that we have the value directly stored in the model we can directly get the object and check/update the status